### ISSUE TITLE
create new workspace angular-nest crashes with angular cli option

### ISSUE THREAD

**Issue by Pekes317** [nrwl/nx#1663]
_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

# Prerequisites

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

Please describe the behavior you are expecting

## Current Behavior

What is the current behavior?

## Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. npx create-nx-workspace projectname or npm init nx-workspace
2. select Angular-Nest (Full Stack)
3. Enter App Name
4. select Angular CLI
5. Crash

### Context

Please provide any relevant information about your setup:

- latest version of NX
-  Angular cli v8.1.2

**A minimal reproduce scenario using allows us to quickly confirm a bug (or point out coding problem) as well as confirm that we are fixing the right problem.**

### Failure Logs

new preset --preset=**nest-angular** --appName=yo --collection=@nrwl/workspace
Cannot parse arguments. See below for the reasons.
    Argument --preset could not be parsed using value **"nest-angular"**. Valid values are: "angular", "react", "web-components", **"angular-nest"**, "react-express", "empty".
(node:24740) UnhandledPromiseRejectionWarning: Error: Command failed: "C:\Users\el_da\AppData\Local\Temp\tmp-247408KsndUr0x2Ph\node_modules\.bin\ng" new preset --preset=nest-angular --appName=yo --collection=@nrwl/workspace
    at checkExecSyncError (child_process.js:631:11)
    at Object.execSync (child_process.js:668:15)
    at createApp (C:\Users\el_da\AppData\Roaming\npm-cache\_npx\24740\node_modules\create-nx-workspace\bin\create-nx-workspace.js:265:21)
    at C:\Users\el_da\AppData\Roaming\npm-cache\_npx\24740\node_modules\create-nx-workspace\bin\create-nx-workspace.js:60:17
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
(node:24740) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:24740) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

## Other

As the error suggests the value being used for the **preset** flag is wrong. NX create workspace is sending **nest-angular** instead of **angular-nest**.

**Comment by ibrahimkafie** [comment]
I have the same issue:
```
Schematic input does not validate against the Schema: {"preset":"nest-angular","appName":"apps","collection":"@nrwl/workspace","name":"ibzoor","style":"scss"}
Errors:

  Data path ".preset" should be equal to one of the allowed values.
```

**Comment by criskrzysiu** [comment]
fix in issue #1675

**Comment by bcabanes** [comment]
Closing the issue since it appears to be fixed.
